tests/functional/s/scenario_tags_pattern_mismatch.feature

```diff
@@ -0,0 +1,31 @@
+Feature: Functional test for missing-required-scenario-tags
+
+    @any_scenario_tag
```

Comment:
Can we include another valid tag to see that the checker handles multiple tags correctly?
Maybe also a second invalid tag would be a good idea - the current implementation loops over all tags, and a potential bug could be that this loop is exited early. Not in this current PR of course, we can see that by reviewing the code, but this could for example happen if someone else refactors this part of the code some time in the future.

Reply:
Added too.